LanguageFeatures/Augmentation-libraries/augmenting_members_A10_t01_lib.dart

```diff
@@ -4,34 +4,19 @@
 
 /// @assertion It is a compile-time error if:
 /// ...
-/// An `external` declaration is augmented with an abstract declaration. For
-/// variables this also applies to the implicit getter and setter.
+/// An `external` variable is augmented with an abstract variable.
 ///
 /// @description Checks that that it is a compile-time error if an `external`
-/// declaration is augmented with an abstract declaration.
+/// variable is augmented with an abstract variable.
```

Comment:
See my comments near the end of https://github.com/dart-lang/language/issues/4017: We might change this.

But currently it doesn't look like the rest of the language team is likely to accept that, so let's just follow the current spec also in this case, and then return and change the test if needed.